presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -169,14 +157,13 @@ private PlanPrinter(
         requireNonNull(plan, "plan is null");
         requireNonNull(types, "types is null");
         requireNonNull(functionRegistry, "functionRegistry is null");
-        requireNonNull(statsProvider, "statsProvider is null");
-        requireNonNull(costProvider, "costProvider is null");
+        requireNonNull(estimatedStatsAndCosts, "estimatedStatsAndCosts is null");
```

Comment:
If you decided to change the name don't forget to fix the message

Reply:
I'm trying to distinguish it from the execution stats.